src/foam/u2/wizard/wizardflow/RemoveWizardlets.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.wizardflow',
+  name: 'EditWizardlet',
```

Comment:
Is this the right name?

Reply:
this flu thing is still affecting me and I hate it